protocol/kafka_segmentio/v2/message_benchmark_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package kafka_segmentio_test
```

Comment:
should it keep same with source files?

Reply:
this is magic test package stuff for go, it means this test is not allowed to access private member functions or internal struct fields but it is allowed to live in the same folder as the source.